### PR TITLE
Default field value

### PR DESCRIPTION
**Pull request by munjaldevelopment**

### CODE REVIEW

**Comment by tabacitu** [comment]
@pxpm can you please review this?

File: src/resources/views/fields/page_or_link.blade.php [hunk 1]
```diff
@@ -13,6 +13,9 @@
     $field['options']['external_link'] = $field['options']['external_link'] ?? trans('backpack::crud.external_link');
 
     $field['pages'] = $field['pages'] ?? ($field['page_model'] ?? config('backpack.pagemanager.page_model_class'))::all();
+
+    $field['internal']['default'] = ($field['default_internal'] ? $field['default_internal'] : "");
```
Comment (pxpm) [inline src/resources/views/fields/page_or_link.blade.php:17]: Why do we change from `$field['default_internal']` to `$field['internal']['default']` ? Are expecting to add more configuration keys insid `internal` ? If not please keep the same `$field['default_internal']`

**Review by pxpm — CHANGES_REQUESTED** [review]

File: src/resources/views/fields/page_or_link.blade.php (hunk 1 above)
Comment (munjaldevelopment) [reply src/resources/views/fields/page_or_link.blade.php:17]: Hi @pxpm please suggest what can be better to do this?

**Comment by jorgetwgroup** [comment]
We will check about this change to merge ASAP.

Thanks

**Comment by pxpm** [comment]
This is now conflicting with base branch. We didn't need this for years, so I suspect this isn't something we really need. 

I am going to close this for now, we can come back and re-evaluate in future if the need arises. 

Thanks @munjaldevelopment 🙏 

Cheers

**Pull request closed without merging**